proto/zitadel/user/v2beta/query.proto

```diff
@@ -184,6 +184,26 @@ message EmailQuery {
     ];
 }
 
+// Query for users with a specific phone.
```

Comment:
Please remove it from v2beta, to not create confusion as it is not used in the SearchQuery message.

Reply:
I synced up v2 and v2Beta, we still rely on v2Beta endpoints.

I would like to keep v2 and v2Beta synced up for this change

Reply:
Works for me, was just a little confused as that there was a definition but no usage.